api/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.7
+
+LABEL Author="Brylie Christopher Oxley"
+LABEL E-mail="brylie.oxley@maas.global"
+LABEL version="0.0.1"
```

Comment:
Could it be more clear to users if we synchronize version with estimator 0.1.2?

Reply:
I think the HTTP API and Python estimator library can have separate release cycles, although I don't have a strong opinion here. @MarkusShepherd, thoughts?

Reply:
Yeah, I've been thinking before how to handling versioning, including tags etc. Maybe it's worth keeping them in sync eventually...

Reply:
OK, let's defer that decision until we have more clarity.